flutter/lib/src/sentry_asset_bundle.dart

```diff
@@ -79,15 +80,15 @@ class SentryAssetBundle implements AssetBundle {
   }
 
   @override
-  Future<T> loadStructuredData<T>(String key, _Parser<T> parser) {
+  Future<T> loadStructuredData<T>(String key, _StringParser<T> parser) {
```

Comment:
Since this is changing the signature from `_Parser` to `_StringParser`, is it compatible with older versions of the Flutter framework? Before the [change](https://github.com/flutter/flutter/pull/119277).
I believe so since this is not available under the `stable` channel yet and CI is green.

Reply:
Tests are passing at least, I'll do a quick check on `stable` to verify

Reply:
Yes, the signature is the same as `_Parser` was -- I renamed it to avoid ambiguity when introducing `_BinaryParser`.